### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by kenygamer**

### CODE REVIEW

File: src/FactionsPro/FactionCommands.php [hunk 1]
```diff
@@ -91,7 +100,7 @@ public function onCommand(CommandSender $sender, Command $command, $label, array
                                         $task = new FactionWar($this->plugin, $r);
                                         $handler = $this->plugin->getServer()->getScheduler()->scheduleDelayedTask($task, 20 * 60 * 2);
                                         $task->setHandler($handler);
-                                        $p->sendMessage("The war against $factionName and $sFaction has started!");
+                                        $p->sendMessage("The war against " . $factionName . " and " . $sFaction . " has started!");
```
Comment (Awzaw) [inline src/FactionsPro/FactionCommands.php]: I think the correct syntax is to use double quotes and enclose variables in curly braces {} where there may be ambiguity. See http://php.net/manual/en/language.types.string.php the section on variable parsing.

Comment (kenygamer) [inline src/FactionsPro/FactionCommands.php]: @Awzaw Mm you may be right, but I prefer to do it that way..

Comment (Awzaw) [inline src/FactionsPro/FactionCommands.php]: If you want to do it like that in your own plugins it's fine, of course; but a PR to  another repo that just changes things to how  you prefer them is not likely to be accepted by anyone, unless you can give a good reason such as performance or readability.

This commit and the one removing the 'use' statements add nothing, and fix nothing.

**Comment by SOF3** [comment]
This is not your own plugin, and we use our own code style.

**Pull request closed without merging**